courses/urls.py

```diff
@@ -0,0 +1,28 @@
+"""Course API URL routes"""
+from django.urls import include, re_path
+from rest_framework import routers
+
+from courses.views import v1
+
+router = routers.SimpleRouter()
+router.register(r"programs", v1.ProgramViewSet, basename="programs_api")
+router.register(r"courses", v1.CourseViewSet, basename="courses_api")
+router.register(r"course_runs", v1.CourseRunViewSet, basename="course_runs_api")
+router.register(
+    r"course_run_certificates",
+    v1.CourseRunCertificateViewSet,
+    basename="course_run_certificates_api",
+)
+router.register(
+    r"program_certificates",
+    v1.ProgramCertificateViewSet,
+    basename="program_certificates_api",
+)
```

Comment:
Until we actually have a request to implement the certificate pages, I think we should get rid of these URL rules. I think it's fine to keep the models and views since the certificate pages will likely be similar to the xpro equivalents, but I don't think those pages should be accessible yet